docs/agents/honeywell_HMR2300.rst

```diff
@@ -0,0 +1,119 @@
+.. highlight:: rst
```

Comment:
This file should be included in the main index file, `index.rst`, under the "Agent Reference" section.